packages/core/src/interfaces/charts.ts

```diff
@@ -174,6 +174,22 @@ export interface PieChartOptions extends BaseChartOptions {
 	};
 }
 
+/**
+ * options specific to gauge charts
+ */
+export interface GaugeChartOptions extends PieChartOptions {
+	gauge?: {
+		center?: {
+			valueFontSize?: Function;
+			deltaFontSize?: Function;
+			titleFontSize?: Function;
+			deltaYPosition?: Function;
+			valueYPosition?: Function;
+			numberFormatter?: Function;
+		};
+		type?: "semi" | "full"
```

Comment:
this should be an ENUM

Reply:
You were anticipated by @ilariaventurini telling me the same :D